### ISSUE TITLE
Does the plugin support kinesis firehose

### ISSUE THREAD

**Issue by sowens-csd** [serverless-operations/serverless-apigateway-service-proxy#83]
I see that Kinesis streams are already supported. Would it work against a Firehose stream? I tried a simple test just replacing a Kinesis stream with a Kinesis Firehose stream and got the error:
```
child "kinesis" fails because ["kinesis" must be an object]
```

I think that probably means that Firehose is not supported?

**Comment by horike37** [comment]
Yes, not supported yet. However, always welcome to add that feature 👍
